### PR TITLE
Fixes issue where job type redirects weren't firing correctly

### PR DESCRIPTION
**Pull request by mabashian**
##### SUMMARY
link #6655 

Here's a gif of the pr in action:

launch

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
 - UI

### CODE REVIEW

**Comment by mabashian** [comment]
recheck

File: awx/ui_next/src/screens/Job/Jobs.jsx [hunk 1]
```diff
@@ -42,6 +41,12 @@ function Jobs({ i18n }) {
     [i18n]
   );
 
+  function TypeRedirect({ view }) {
+    const { id } = useParams();
+    const { path } = useRouteMatch();
```
Comment (marshmalien) [inline awx/ui_next/src/screens/Job/Jobs.jsx:46]: On line 22, you could replace `const match = useRouteMatch();` with this line, and replace every instance of `match.path` with `path`.

**Review by marshmalien — APPROVED** [review]
Tested this out locally and it redirected as expected.

**Comment by jakemcdermott** [comment]
recheck

**Comment by dsesami** [comment]
It works!

**Review by dsesami — APPROVED** [review]

**Pull request merged**